src/include/ipc4/base-config.h

```diff
@@ -23,9 +23,8 @@
 
 #ifndef __SOF_IPC4_BASE_CONFIG_H__
 #define __SOF_IPC4_BASE_CONFIG_H__
-#ifndef __SOF_MODULE_SERVICE_BUILD__
+
 #include <sof/compiler_attributes.h>
-#endif
```

Comment:
This too, is this removing more?

Reply:
@kv2019i no, all those changes were brought in with the PR, that enabled modular smart-amp-test https://github.com/thesofproject/sof/pull/8580/commits/694aa9e3feb2352296cdebfb0ed1eec594e18d9a

Reply:
@lyakh But smart-ampt-test was an example, wouldn't these changed be needed by other LMDK modules? OTOH, the CI tests seem to pass (there's at least the mixer module), so maybe these were needed just for smart-amp-test and not LMDK support in general. Just not so easy to make this conclusion from the diff.

Reply:
@kv2019i that's also my understanding, that since these changes were introduced with the smart-amp-test and reverting them doesn't break QB tests, don't think they were needed for other LMDK examples, i.e. for the down-mixer